client/src/app/hero-header/hero-header.component.scss

```diff
@@ -0,0 +1,3 @@
+op-hero-header {
+  background-color: ##f0f0f0;
+}
```

Comment:
Yeah I forget about that, but I'm actually intend to delete the SCSS file, it's just for testing, still WIP..